frontend/routes/ConfigRoute.tsx

```diff
@@ -12,12 +12,15 @@ interface ConfigRouteProps {
 }
 
 export class ConfigRoute extends React.Component<ConfigRouteProps> {
-	public componentDidMount() {
+	public async componentDidMount() {
 		this.props.dispatch(BreadcrumbActions.setBreadcrumb('Config'));
-
-		// if (this.props.games.size === 0) {
-		fetchGamesData(this.props.dispatch);
-		// }
+		try {
+			const gamesData = await ApiRequests.get<Array<Game>>('games', {});
+			this.props.dispatch(DataLoadActions.setGames(gamesData));
+			this.props.dispatch(GameVisibilityActions.setEveryGameVisibility(true));
+		} catch (error) {
+			console.log(error);
+		}
 	}
 	public render() {
 		const changeGamesVisibilityCallback = (allVisible: boolean) => {
```

Comment:
While I am thinking about it, please refector this function out to not be recreated each render, do it like explained here in section "Arrow function x2" https://typeofweb.com/poprawne-bindowanie-funkcji-react-js/